### PR TITLE
Add R12/R13 Set for Hunter

### PR DESCRIPTION
**Pull request by Kawney**
This is my second attempt to try and provide to this project; I am still rather inexperienced and unsure of what I'm doing, BUT here I am attempting to add both factions PvP set bonuses, linked here: https://www.wowhead.com/classic/item-set=1725/warlords-pursuit https://www.wowhead.com/classic/item-set=1726/warlords-prowess https://www.wowhead.com/classic/item-set=1739/field-marshals-pursuit https://www.wowhead.com/classic/item-set=1738/field-marshals-prowess

If there's any issues / errors with these changes, assistance may be necessary / extremely appreciated!

### CODE REVIEW

File: sim/hunter/item_sets_pvp.go [hunk 1]
```diff
@@ -128,3 +128,99 @@ var ItemSetLieutenantCommandersProwess = core.NewItemSet(core.ItemSet{
...
+})
+
+var ItemSetWarlordsProwess = core.NewItemSet(core.ItemSet{
+	Name: "Warlord's Prowess",
+	Bonuses: map[int32]core.ApplyEffect{
+		// +20 Agility.
+		2: func(agent core.Agent) {
+			c := agent.GetCharacter()
+			c.AddStat(stats.Agility, 20)
+		},
```
Comment (kayla-glick) [inline sim/hunter/item_sets_pvp.go:169]: Looks like Warlord's Prowess also increases stam by 20
image

File: sim/hunter/item_sets_pvp.go [hunk 2]
```diff
@@ -128,3 +128,99 @@ var ItemSetLieutenantCommandersProwess = core.NewItemSet(core.ItemSet{
 		},
 	},
 })
+
+///////////////////////////////////////////////////////////////////////////
+//                            SoD Phase 5 Item Sets
+///////////////////////////////////////////////////////////////////////////
+
+var ItemSetWarlordsPursuit = core.NewItemSet(core.ItemSet{
```
Comment (kayla-glick) [inline sim/hunter/item_sets_pvp.go:136]: It looks like the gloves of the Pursuit sets have +4% damage to multi-show as well. That can be done in `sim/hunter/items.go` for both pairs
image

File: sim/hunter/item_sets_pvp.go [hunk 3]
```diff
@@ -128,3 +128,99 @@ var ItemSetLieutenantCommandersProwess = core.NewItemSet(core.ItemSet{
...
+			c := agent.GetCharacter()
+			c.AddStats(stats.Stats{
+				stats.Agility:           20,
+				stats.Spellpower:        23,
+			})
+		},
+	},
+})
+
+var ItemSetWarlordsProwess = core.NewItemSet(core.ItemSet{
```
Comment (kayla-glick) [inline sim/hunter/item_sets_pvp.go:162]: The Prowess sets also have a +4% raptor strike damage mod on the gloves if you wouldn't mind adding those in `sim/hunter/items.go` too
image

**Review by kayla-glick — COMMENTED** [review]
Thanks for contributing! Just a couple things but always appreciate getting new folks involved!

File: sim/hunter/item_sets_pvp.go (hunk 3 above)
Comment (Kawney) [reply sim/hunter/item_sets_pvp.go:162]: I believe both this pair and the other pair are included in this pull from Docoda:
https://github.com/wowsims/sod/pull/1001

Comment (kayla-glick) [reply sim/hunter/item_sets_pvp.go:162]: That's totally fine then! I know @ncberman is taking a look at Docoda's PR to get it cleaned up and passing so we'll get them there!

**Review by kayla-glick — APPROVED** [review]

**Pull request merged**